app/containers/ExplorePage.tsx

```diff
@@ -192,6 +229,9 @@ export default function ExplorePage() {
 
   // eslint-disable-next-line
   const filename = (filePath !== undefined) ? filePath.replace(/^.*[\\\/]/, '') : "";
+  const overridesCount = t('explore.overrides', {
+    count: observations ? observations.reduce((a, b) => a + (b.pred_1 !== b.label ? 1 : 0), 0) : 0
```

Comment:
I think it will be more readable if you put the expressions for `count` in a helper function, and perhaps inline the variable then.

Reply:
fixed